examples/dynamic-imports/static.config.js

```diff
@@ -1,5 +1,27 @@
+import React from 'react'
 import axios from 'axios'
-//
+import fs from 'fs'
+import path from 'path'
+import webpack from 'webpack'
+import { ReportChunks } from 'react-universal-component'
+import flushChunks from 'webpack-flush-chunks'
+
+const resolve = p => path.resolve(__dirname, p)
+const nodeModules = resolve('./node_modules')
+
+// for SSR of dynamic imports
+const externals = fs
```

Comment:
I’m not entirely sure why we can’t use webpack- node-externals but we could move this function to core as well

Reply:
I found this pattern digging in the issues of react-universal-component

Reply:
We could just add a config that says whitelist external node modules?